fivetran_provider_async/example_dags/example_fivetran_xcom.py

```diff
@@ -0,0 +1,40 @@
+import time
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+
+from fivetran_provider_async.operators import FivetranOperator
+from fivetran_provider_async.sensors import FivetranSensor
+
+default_args = {
+    "owner": "Airflow",
+    "start_date": datetime(2021, 4, 6),
+    "provide_context": True,
+}
+
+dag = DAG(
+    dag_id="example_fivetran_xcom",
+    default_args=default_args,
+    schedule_interval=timedelta(days=1),
+    catchup=False,
+)
+
+with dag:
+    fivetran_operator = FivetranOperator(
+        task_id="fivetran-operator",
+        fivetran_conn_id="fivetran_default",
+        connector_id="{{ var.value.connector_id }}",
```

Comment:
How are we creating airflow variables? I think they are vital to running these dags, should we explain how to create/get them?

Reply:
manual testing is done on Astro cloud

Reply:
we havent automated it

Reply:
yes, we have manual testing. I have attached the dagrun screenshot